tests/angular/services/API.service.spec.js

```diff
@@ -0,0 +1,45 @@
+ngDescribe({
+    name: 'Test API service',
+    modules: 'app',
+    tests: function (API) {
+        var httpBackend, q, scope;
+        beforeEach(inject(function( _$httpBackend_, $q, $rootScope) {
+            q = $q;
+            httpBackend=_$httpBackend_;
+            scope = $rootScope.$new();
+        }));
+        function getValue(promise,q,scope){
+                var defer = q.defer();
+                var unproxiedPromise;
+                promise.then(function(value){
+                    unproxiedPromise = value;
+                });
```

Comment:
I think it's a better idea to test "our" configuration of the `API` service rather than testing restangular..  as in:
* constructed url must start with `/api/` (prefix)
* token must be sent
* etc..

but we can keep this for later on so I think we can just omit testing the API service for now